App/Navigation/AppNavigation/Primary/TabNavigator/index.tsx

```diff
@@ -41,4 +41,9 @@ const nav = createBottomTabNavigator(
   }
 )
 
+nav.navigationOptions = {
+  // Hide the header from AppNavigator stack
+  header: null
```

Comment:
When the tab navigator is displayed on the stack, hide the stack's nav bar because each of the tabs holds a stack navigator that displays a nav bar.